ch2driver/pytpcc/tpcc.py

```diff
@@ -253,6 +253,15 @@ def executorFunc(clientId, TAFlag, driverClass, qDone, warmupDurationQ, warmupDu
                          help='Enable loading the data through the data service')
     aparser.add_argument('--qrysvc-load', action='store_true',
                          help='Enable loading the data through the query service')
+    aparser.add_argument('--ch2p', action='store_true', help='Create CH2+ schema')
+    aparser.add_argument('--ch2pp', action='store_true', help='Create CH2++ schema')
+    aparser.add_argument('--customerExtraFields', default=32, type=int,
```

Comment:
We could set the default values to the new constants you added in `constants.py`.

E.g.
```
aparser.add_argument('--customerExtraFields', default=constants.CH2PP_CUSTOMER_EXTRA_FIELDS, type=int, help='Number of extra unused fields in Customer')
```

Reply:
Done.